core-bundle/src/Controller/Page/AbstractLayoutPageController.php

```diff
@@ -75,6 +79,10 @@ public static function getSubscribedServices(): array
 
     protected function getResponseContext(PageModel $page): ResponseContext
     {
+        if ($responseContext = $this->container->get('contao.routing.response_context_accessor')->getResponseContext()) {
```

Comment:
this seems to be unrelated to ehe PR title?

Reply:
Yes please. Unrelated changes in a PR make them much harder to track.

Reply:
Done 👍